frontend/dashboard/pages/PageLayout/DashboardHeader/SmallHeaderMenu/SmallHeaderMenu.test.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import { SmallHeaderMenu } from './SmallHeaderMenu';
+import { textMock } from '@studio/testing/mocks/i18nMock';
+import userEvent from '@testing-library/user-event';
+import { headerContextValueMock } from '../../../../testing/headerContextMock';
+import { HeaderMenuGroupKey } from '../../../../enums/HeaderMenuGroupKey';
+import { HeaderContext, type HeaderContextProps } from '../../../../context/HeaderContext';
+import { renderWithProviders } from '../../../../testing/mocks';
+import { addFeatureFlagToLocalStorage, FeatureFlag } from 'app-shared/utils/featureToggleUtils';
+
+describe('SmallHeaderMenu', () => {
+  beforeAll(() => {
+    addFeatureFlagToLocalStorage(FeatureFlag.OrgLibrary);
+  });
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should render the menu trigger button with the correct text', () => {
+    renderSmallHeaderMenu();
+
+    expect(screen.getByRole('button', { name: textMock('top_menu.menu') })).toBeInTheDocument();
+  });
+
+  it('should open the dropdown menu when the trigger button is clicked', async () => {
+    const user = userEvent.setup();
+    renderSmallHeaderMenu();
+
+    expect(
+      screen.queryByRole('menuitem', {
+        name: textMock(headerContextValueMock.menuItems[0].key),
+      }),
+    ).not.toBeInTheDocument();
+
+    const button = screen.getByRole('button', { name: textMock('top_menu.menu') });
+    await user.click(button);
+
+    expect(
+      screen.getByRole('menuitem', {
+        name: textMock(headerContextValueMock.menuItems[0].key),
+      }),
+    ).toBeInTheDocument();
+  });
+
+  it('should close the dropdown menu when an item is clicked', async () => {
+    const user = userEvent.setup();
+    renderSmallHeaderMenu();
+
+    const button = screen.getByRole('button', { name: textMock('top_menu.menu') });
+    await user.click(button);
+
+    const menuItem = screen.getByRole('menuitem', {
+      name: textMock(headerContextValueMock.menuItems[0].key),
+    });
+    expect(menuItem).toBeInTheDocument();
+    await user.click(menuItem);
+
+    expect(
+      screen.queryByRole('menuitem', {
+        name: textMock(headerContextValueMock.menuItems[0].key),
+      }),
+    ).not.toBeInTheDocument();
```

Comment:
There's no need to query for the menu item twice.

```suggestion
    const menuItem = screen.getByRole('menuitem', {
      name: textMock(headerContextValueMock.menuItems[0].key),
    });
    expect(menuItem).toBeInTheDocument();
    await user.click(menuItem);

    expect(menuItem).not.toBeInTheDocument();
```